apps/snitch_core/test/data/schema/rating/rating_option_test.exs

```diff
@@ -0,0 +1,47 @@
+defmodule Snitch.Data.Schema.RatingOptionTest do
+  use ExUnit.Case, async: true
+  use Snitch.DataCase
+  import Snitch.Factory
+  alias Snitch.Data.Schema.RatingOption
+
+  setup :rating_options
+
+  @tag rating_option_count: 1
+  describe "create_changeset/2" do
+    test "returns a valid changeset", %{rating_options: [rating_option]} do
+      rating_option = Map.from_struct(rating_option)
```

Comment:
This is not the right approach. Inserting the `rating_options` and then using the same. Please do it with params.